build.gradle

```diff
@@ -0,0 +1,278 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java application project to get you started.
+ * For more details take a look at the 'Building Java & JVM projects' chapter in the Gradle
+ * User Manual available at https://docs.gradle.org/7.4.2/userguide/building_java_projects.html
+ */
+
+plugins {
+	id 'java'
+	id 'war'
+	id 'jacoco'
+	id 'org.ajoberstar.grgit' version '4.1.1'
+}
+
+repositories {
+	mavenCentral()
+}
+// Warning - these values are hard coded in AuthController.java
+def JAR_TEMPLATE_DIR = 'kbase_auth2_templates'
+def TEMPLATE_LIST_FILE_NAME = "templates.manifest"
+
+task buildGitCommitFile {
+	doLast {
+		def commitId = grgit.head().id
+		// is there a varible for builddir/classe/java/main?
```

Comment:
typo

Reply:
I have these fixed locally, they'll show up in a future PR if that's ok